subsys/suit/orchestrator/src/suit_orchestrator_sdfw.c

```diff
@@ -348,9 +347,20 @@ static int boot_path(bool emergency)
 	}
 
 	for (size_t i = 0; i < class_ids_to_boot_len; i++) {
-		ret = boot_envelope((const suit_manifest_class_id_t *)class_ids_to_boot[i]);
+		const suit_manifest_class_id_t *class_id =
+			(const suit_manifest_class_id_t *)class_ids_to_boot[i];
+
+		ret = boot_envelope(class_id);
 		if (ret != 0) {
-			LOG_ERR("Booting %d manifest failed (%d)", i, ret);
+			LOG_ERR("Booting %d manifest failed (%d):", i, ret);
```

Comment:
Would it be a good idea to log out total amount of manifests in boot order and number of manifest, i.e. "Booting manifests 1/2 failed (error code)",  "Booting manifests 2/2 failed (error code)"